packages/thumbprint-react/components/DatePicker/index.jsx

```diff
@@ -46,7 +47,8 @@ export default function DatePicker(props) {
                 disabledDays={disabledDays}
                 fromMonth={get(disabledDays, 'before', null)}
                 toMonth={lastMonth}
-                initialMonth={selectedDays[0]}
+                month={month || selectedDays[0]}
```

Comment:
Do we need `selectedDays` here? Would be simpler to just make both `month` and `onMonthChange` required?

Reply:
It reduces boilerplate for the consumer. This way I can initialise `month` as `undefined`, and it will default to showing the month for the first date in `value`. Making it required means that I have to pass in the initial value twice, effectively.

Reply:
Ah, I see what you mean. Sounds good!